### PR TITLE
Add Installer type 

### PR DESCRIPTION
**Pull request by Reset12138**
According to [#695](https://github.com/microsoft/winget-pkgs/pull/695)
More installer types are supported by winget

### CODE REVIEW

File: src/WinGetYamlGenerator/InstallerInfo.cs [hunk 1]
```diff
@@ -17,6 +17,10 @@ public enum InstallerKind
         EXE,
         MSI,
         MSIX,
+        INNO,
+        WINX,
```
Comment (ptorr-msft) [inline src/WinGetYamlGenerator/InstallerInfo.cs:21]: Typo - `WIX`?

File: src/WinGetYamlGenerator/InstallerInfo.cs [hunk 2]
```diff
@@ -17,6 +17,10 @@ public enum InstallerKind
...
+        NULLSOFT,
+        APPX,
```
Comment (ptorr-msft) [inline src/WinGetYamlGenerator/InstallerInfo.cs:23]: Please put `APPX` directly after `MSIX`.

File: src/WinGetYamlGenerator/InstallerInfo.cs (hunk 1 above)
Comment (Reset12138) [reply src/WinGetYamlGenerator/InstallerInfo.cs]: > Thanks for the contribution! See comments in the file, and also please update the comment in `InstallerKindValueConverter.cs` and also the drop-down in `MainPage.xaml`

got it. I will update the others file later

**Comment by Reset12138** [comment]
My pleasure.


-------- 原始信息 --------
发件人： "Peter Torr (MSFT)" <notifications@github.com>
日期: 2020/5/27 01:32 (GMT+08:00)
收件人： ptorr-msft/WinGetYamlGenerator <WinGetYamlGenerator@noreply.github.com>
抄送： 谭源 <kentertan12138@Outlook.com>, Author <author@noreply.github.com>
主题： Re: [ptorr-msft/WinGetYamlGenerator] Add Installer type  (#17)


@ptorr-msft approved this pull request.

Thanks!

―

**Pull request merged**